scripts/internal/manylinux-build-common.sh

```diff
@@ -12,7 +12,7 @@ if [[ $# -eq 0 ]]; then
   PYBIN=(/opt/python/*/bin)
   PYBINARIES=()
   for version in "${PYBIN[@]}"; do
-    if [[ ${version} == *"cp37"* || ${version} == *"cp38"* || ${version} == *"cp39"* || ${version} == *"cp310"* ]]; then
+    if [[ ${version} == *"cp37"* || ${version} == *"cp38"* || ${version} == *"cp39"* || ${version} == *"cp310"* || ${version} == *"cp311" ]]; then
```

Comment:
Note: #212 likely also brings in new CPython 3.11 into the docker images

Reply:
@thewtex As noted below the updated docker image does include Python 3.11 but it is a prerelease from a few months before the actual 3.11 release. Any thoughts on whether this needs to be updated in the near term?

Reply:
Pulling the latest manylinux images, it looks like they now have 3.11.0 -- I will work on pushing an updated image.

Reply:
Thank you!